File: 백준/Bronze/2562. 최댓값/최댓값.cc
#include <iostream>
using namespace std;
int main(void)
{
    int a[9];
    int max=0;
    int maxindex=0;
    for(int i=0;i<9;i++)
    {
        cin>>a[i];
        if(a[i]>max){
            max=a[i];
            maxindex=i+1;
        }
    }
    cout<<max<<endl;
    cout<<maxindex;
}

File: 백준/Bronze/2562. 최댓값/최댓값.java
import java.util.Scanner;
public class Main{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        int[] a=new int[9];
        int max=0;
        int maxindex=0;
        for(int i=0;i<9;i++)
        {
            a[i]=sc.nextInt();
            if(a[i]>max){
                max=a[i];
                maxindex=i+1;
            }
        }
    System.out.println(max);
    System.out.println(maxindex);
    }
}